torch/distributed/distributed_c10d.py

```diff
@@ -918,14 +927,14 @@ def is_backend_available(backend: str) -> bool:
 
 
 def is_initialized() -> bool:
-    """
-    Checking if the default process group has been initialized
-    """
+    """Check if the default process group has been initialized."""
     return GroupMember.WORLD is not None
 
 
 def is_torchelastic_launched() -> bool:
     """
+    Check process launch.
+
     Checks whether this process was launched with ``torch.distributed.elastic``
     (aka torchelastic). The existence of ``TORCHELASTIC_RUN_ID`` environment
     variable is used as a proxy to determine whether the current process
```

Comment:
Maybe:
```suggestion
    Check whether this process was launched with ``torch.distributed.elastic`` (aka torchelastic). 
    
    The existence of ``TORCHELASTIC_RUN_ID`` environment
    variable is used as a proxy to determine whether the current process
```

Reply:
@svekars thanks so much for the suggestion and yes, but few other places the description is longer. (e.g. def get_rank in LOC 1465) The summary needs to be one liner so ideally we want the best possible short one-line summary and the description can provide the details as described in the original doc strings. Let me try re-write summary here and at few other places that @H-Huang commented and update the PR.